Tasks/NuGetCommand/nugetcommandmain.ts

```diff
@@ -52,4 +55,81 @@ async function main(): Promise<void> {
     }
 }
 
+
+function _logNugetStartupVariables() {
+    try {
+        let externalendpoint = null;
+        let epId = tl.getInput('externalendpoint');
+        if (epId) {
+            externalendpoint = {
+                feedName: tl.getEndpointUrl(epId, true).replace(/\W/g, ''),
+                feedUri: tl.getEndpointUrl(epId, true)
+            }
+        }
+        let externalendpoints = tl.getDelimitedInput('externalendpoints', ',');
+        if (externalendpoints) {
+            externalendpoints = externalendpoints.reduce((ary, id) => {
+                let te = {
+                    feedName: tl.getEndpointUrl(id, true).replace(/\W/g, ''),
+                    feedUri: tl.getEndpointUrl(id, true)
+                }
+                ary.push(te);
+                return ary;
+            }, []);
+        }
+        let nugetTelemStr = JSON.stringify(
+            {
+                'command': tl.getInput('command'),
```

Comment:
Curious, what was the reasoning behind a one for all telemetry approach instead of logging it depending on the scenario? I.E. why is this here and not in every command?

Reply:
Simplicity.